src/js/file-manager.js

```diff
@@ -163,4 +163,50 @@ fileManager.isTooLarge = () => { return false; };
  */
 fileManager.getMaxSizeReadable = () => { return `${5}MB`; };
 
+/**
+ * @function resizeImage
```

Comment:
We actually create a replacement for `file-manager.js` in Enketo Express. So let's move this method elsewhere. I'm thinking in `utils.js`.

Reply:
`resizeImage` is moved into `utils.js`